### PR TITLE
chore: Improved developer experience by adding development commands to composer.json scripts

### PR DESCRIPTION
**Pull request by norberttech**
This is a bit opinionated approach so feel free to close that proposal. 
From my experience simplifying the developer experience increases the number of valuable contributions.
I believe that exposing development tools through composer scripts makes devs' work easier since there is only one place they need to look to understand how to work with that repo.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -74,7 +74,7 @@ be found for Mac or Windows.
 3.  Run tests.
 
     ```sh
-    > vendor/bin/phpunit
+    > php composer.phar test
```
[the remark this answers is not in the document]
Comment (norberttech) [reply README.md:77]: I dont believe anyone does that but this is how it was done before so I thought its maybe a preferred approach. Will change that tomorrow

**Review by bshaffer — APPROVED** [review]
This is great, thank you for this update!

**Comment by bshaffer** [comment]
Now that we have these commands, we can update our [workflow file](https://github.com/googleapis/gax-php/blob/main/.github/workflows/tests.yml) to match!

**Comment by norberttech** [comment]
Speaking about workflow files, what are your thoughts about running all the tools (coding standards / static analysis) on the lowest supported version of PHP with the latest versions of dependencies? 

So let's say this lib supports PHP: `[8.0, 8.1, 8.2]`, I would change static analysis and coding standards to run on `8.0`. 

I ran into some discrepancies in the past between CI and people's local development environments. 
After that, I made a rule for all my repos that development should always happen on the lowest supported version of PHP to reduce the possibility of using unsupported features or getting false positive results from static analysis.

**Pull request merged**